recipe/meta.yaml

```diff
@@ -36,19 +36,20 @@ requirements:
   run:
     - python
     - bleach
-    - bokeh >=2.4.0,<2.5.0
+    - packaging
+    - bokeh >=3.1.1,<3.2
     - markdown
     - param >=1.12.0
     - pyct >=0.4.4
```

Comment:
`pyct` isn't mentioned in `install_requires` https://github.com/holoviz/panel/blob/15acb1b148cf1a6ea39bf19670dc281d1bac7f03/setup.py#L103-L117
Is it still required?

Reply:
It is not, good catch!